src/int_to_bytes.rs

```diff
@@ -0,0 +1,25 @@
+pub fn u64_to_bytes(n: u64) -> Vec<u8> {
+    let mut buf = Vec::<u8>::new();
+    buf.extend_from_slice(&n.to_be_bytes());
```

Comment:
Nitpicky, but you could just go `if n < 0 buf.push(0);`

Reply:
`n` is unsigned, so it would have to be `if n as i64 { buf.push(0); }` which would make me a bit nervous about whether rust checks for overflow in casts like that.. I suppose it doesn't